### PR TITLE
Send colored logs to non-tty destinations + fix for a unit test

### PR DESCRIPTION
**Pull request by sb98052**
Add option to ConsoleRenderer for sending colored logs to non-tty destinations. 
The main use case is that of files meant to be streamed to the console. 
We use this with docker logs.
Also included is a fix for a unit test.

### CODE REVIEW

**Comment by hynek** [comment]
This looks useful however CI is failing (I think you just have to split a line) and it needs a changelog entry.

thanks!

(p.s. I’ll be off the grid in the next two weeks so it might take a while until I'll be able to re-review.)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ Changes:
 - Empty strings are valid events now.
   `#110 <https://github.com/hynek/structlog/issues/110>`_
 
+- `structlog.dev.ConsoleRenderer`` now accepts a *force_colors* argument to output colored logs even if the destination is not a tty. Use this option if your logs are stored in files that are intended to be streamed to the console.
```
Comment (hynek) [inline CHANGELOG.rst]: - You're missing a backtick here.
- We use [semantic newlines](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) in our reST docs.

File: tests/test_dev.py [hunk 2]
```diff
@@ -228,6 +228,30 @@ def test_colorama_colors_false(self):
         assert dev._PlainStyles is plain_cr._styles
         assert "[info     ] event                          foo=bar" == rv
 
+    @pytest.mark.skipif(not dev._has_colorama,
+                        reason="Colorama must be missing.")
```
Comment (hynek) [inline tests/test_dev.py]: This doesn't seem right?

**Comment by sb98052** [comment]
Thanks for the review. It appears that CI is broken even without this change set. I debugged the issue and have submitted a proposed fix as part of this pull request.

**Comment by hynek** [comment]
Hi, I’m super sorry it took so long!  Thank you for your contribution!

**Pull request merged**